bin/index.js

```diff
@@ -49,7 +51,7 @@ try {
   main = require(path.join(__dirname, '..', 'commands', `${command}.js`))
 } catch (err) {
   console.error(chalk.red(`
-Unknown command: ${command}`))
+Unknown command: ${command}`, err))
```

Comment:
We should aim to never expose customers to stack traces and other raw errors.

Perhaps we should follow a pattern seen elsewhere and have a `--debug` flag that enables this sort of thing?